SwappinessCheck.pm

```diff
@@ -49,7 +49,7 @@ sub execute
 		if ($1 > 10)
 		{
 			$self->{COMMENT} = "** vm.swappiness is larger than 10\n" .
-				"set it with '/sbin/sysctl -w vm.swappiness=10'\n".
+				"set it by adding to /etc/sysctl vm.swappiness=10\n".
```

Comment:
This looks good to me :+1:

Reply:
Perhaps clarify that you need to reboot for that setting to take effect? Or is there another way to make sure of that?